scripts/validate-version-strategy.mjs

```
@@ -0,0 +1,149 @@
+        )
+        hasErrors = true
+      } else {
+        console.log(`  ✅ Publish config correct`)
+      }
+    }
+
+    console.log()
+  }
+
+  if (hasErrors) {
+    console.log('❌ Version strategy validation failed')
```

ESLint

[error] 'console' is not defined.

(no-undef)